src/met-atlas-viewer.js

```diff
@@ -496,51 +457,50 @@ function MetAtlasViewer(targetElement) {
         // copy camera to move it around without having to mess with the scene,
         // but make the FoV slightly narrower to make sure things are inside the
         // visible portion of the view on the main camera.
-        let testCamera = new PerspectiveCamera(fieldOfView-10, aspect, near, far);
-        testCamera.position.copy(t);
-        testCamera.up.copy({x:0, y:1, z:0});
-        testCamera.lookAt(0,0,0);
-        testCamera.updateMatrix();
-        testCamera.updateMatrixWorld();
-        testCamera.matrixWorldInverse.copy(testCamera.matrixWorld).invert();
+        let testCamera = new PerspectiveCamera(
+          fieldOfView - 10,
+          aspect,
+          near,
+          far
+        );
+        moveCamera(testCamera, t);
 
         while (!isInCamera(items, testCamera) && cameraDistance < maxDistance) {
           // move camera back until we've reached the max distance or we can see
           // all object in the camera
           cameraDistance += cameraStep;
           t = {
-            x: p.x + p.x/l * cameraDistance,
-            y: p.y + p.y/l * cameraDistance,
-            z: p.z + p.z/l * cameraDistance
+            x: p.x + (p.x / l) * cameraDistance,
```

Comment:
There are lots of this kind of linear mapping of vectors, maybe worth a function? Since in all use cases, the transformation matrix is a uniform scaling matrix, the function requires just one parameter.

Reply:
I think this is a good candidate for refactoring. I see two instances of it in the `focusOnItems` function and a few more similar ones for example in `flyUpdate`. Adding a function that works for all of these would require more parameters though. Currently I feel split on if this function should be added or if we should leave things as is since they are relatively concise but still different a bit in different instances.

Reply:
Fair enough. I'm OK to leave it as is.